lib/ruby_lsp/executor.rb

```diff
@@ -169,6 +169,8 @@ def run(request)
         completion(uri, request.dig(:params, :position))
       when "textDocument/definition"
         definition(uri, request.dig(:params, :position))
+      when "textDocument/showSyntaxTree"
```

Comment:
Is there any convention to distinguish custom requests from standard requests? I think I saw some LSPs use an underscore prefix.

Reply:
I like the idea of a prefix to avoid future conflicts 👍

Reply:
Added `rubyLsp/` as the prefix for custom requests.